src/igvfd/schemas/software_version.json

```diff
@@ -76,5 +76,9 @@
             "format": "uri"
         }
     },
+    "fuzzy_searchable_fields": [],
```

Comment:
why do we need empty list of fuzzy?

Reply:
The reasoning for adding empty lists in schema definitions is so that it is clear where we would add the needed items.  Before we had only one list and so every type had @type in the list.  Now we don't have a field that shows up everywhere.  I am suggesting that should be summary.  So  I would not bother to remove the empty list as we will likely be adding and subtracting from it

Reply:
agree with empty lists as a pointer is a good idea